Jenkinsfile

```
@@ -1,33 +1,58 @@
 pipeline {
-  agent {
-    docker {
-      image 'node:10-alpine'
-    }
+  agent { 
+    label 'linux'
```

Why not `linux && immutable`?

---

It can be as specific as you wish. I used linux just for a quick PR, but sure you can add `linux && immutable `, let me do it now

```suggestion
    label 'linux && immutable'
```